src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -117,6 +117,21 @@ typedef NS_ENUM(NSUInteger, MTRDeviceExpectedValueFieldIndex) {
     MTRDeviceExpectedValueFieldIDIndex = 2
 };
 
+typedef NS_ENUM(NSUInteger, MTRDeviceReadRequestFieldIndex) {
+    MTRDeviceReadRequestFieldEndpointIDIndex = 0,
+    MTRDeviceReadRequestFieldClusterIDIndex = 1,
+    MTRDeviceReadRequestFieldAttributeIDIndex = 2,
+    MTRDeviceReadRequestFieldParamsIndex = 3
+};
+
+typedef NS_ENUM(NSUInteger, MTRDeviceWorkItemBatchingID) {
+    MTRDeviceWorkItemBatchingReadID = 1,
```

Comment:
Shouldn't that normally be MTRDeviceWorkItemBatchingIDRead?

Not that this matters too much, since this is all internal.

Reply:
I actually struggle with this. I've seen different ways of doing this, including "PREFIXBlahErrorSomethingError".

Nowadays I try to sound it out and in the moment `MTRDeviceWorkItemBatchingReadID` felt better than `MTRDeviceWorkItemBatchingIDRead` to me. I can certainly move this around.